openapiart/bundler.py

```diff
@@ -1060,6 +1060,68 @@ def _generate_value_schema(
                 metric_tags_schema_name
             ] = metric_tags_schema
 
+        if "features" in xpattern and "random" in xpattern["features"]:
+            if xpattern["format"] in ["integer", "ipv4", "ipv6", "mac"]:
+                random_pattern_name = "{}.Random".format(schema_name)
+                schema["properties"]["choice"]["x-enum"]["random"] = {
+                    "x-field-uid": 6
+                }
+                schema["properties"]["random"] = {
+                    "$ref": "#/components/schemas/{}".format(
+                        random_pattern_name
+                    )
+                }
+                schema["properties"]["random"]["x-field-uid"] = auto_field.uid
+                random_auto_field = AutoFieldUid()
+                random_schema = {
+                    "description": "{} random pattern".format(
+                        xpattern["format"]
+                    ),
+                    "type": "object",
+                    "properties": {
```

Comment:
Should we have options to exclude loopback, link-local, and/or mutlicast addresses when either IP format is chosen?

Reply:
well this is a generic Idea should not the person building model be aware and not include those fields with random support ?

Reply:
If we have such requirement, model can be extended to add such exclusion list without breaking backward compatibility. openapiart can be extended to support such exclusion specific to a field format. Assumption, backend implementation (UHD/OTG) can also support such exclusion.